addon/components/base/bs-form/element.js

```diff
@@ -330,7 +330,7 @@ const nonDefaultLayouts = A([
  */
 export default FormGroup.extend({
   layout,
-  classNameBindings: ['disabled:is-disabled', 'required:is-required', 'isValidating'],
+  classNameBindings: ['disabled:disabled', 'required:is-required', 'isValidating'],
```

Comment:
I changed it to `disabled` instead of `is-disabled`, because BS4 and BS3 both add styling (like a `cursor` etc.) to the `label` when using the `disabled` class (see for example https://v4-alpha.getbootstrap.com/components/forms/#checkboxes-and-radios).

But you're right it isn't worth a breaking change. I guess it would be better to keep it `is-disabled` and let the users add the styling themselves?